src/qililab/instruments/qblox/qblox_d5a.py

```diff
@@ -85,7 +84,11 @@ def setup(self, parameter: Parameter, value: float | str | bool, channel_id: int
                 f"the specified dac index:{channel_id} is out of range."
                 + " Number of dacs is 4 -> maximum channel_id should be 3."
             )
-        channel = self.dac(dac_index=channel_id)
+        if self.is_device_initialized():
+            channel = self.dac(dac_index=channel_id)
+        else:
+            channel = None
+
```

Comment:
won't this set the channel to `None` on instrument initialization?

Reply:
What do you mean by instrument initialization? `initial_setup`?